style.css

```diff
@@ -347,3 +354,223 @@ footer {
   font-size: 22px;
   list-style-type: none;
 }
+
+/* ===== Desktop Version ===== */
+
+@media (min-width: 768px) {
+  /* ===== Navbar Section ===== */
+  .navBar {
+    padding: 0 142px 0 130px;
+    align-items: center;
+    height: 72px;
+  }
```

Comment:
- [ ] Please make sure your logo is in the same font family as the design.
- [ ] Also, kindly confirm that you are using the right font-sizes and spacing in the nav section